debug/gdbdiff.ts

```diff
@@ -23,25 +63,31 @@ function compareRegisters(emulator: Uint32Array, silicone: Uint32Array) {
 async function main() {
   const client1 = new GDBClient();
   const client2 = new GDBClient();
-  await client1.connect('localhost');
-  await client2.connect('raspberrypi');
+  await client1.connect('localhost', 3334);
+  await client2.connect('localhost', 3333);
   // Disable interrupts
   await client1.writeRegister(19, 1, 8);
   await client2.writeRegister(19, 1, 8);
   // Start diffing
-  for (let counter = 0; ; counter++) {
+  let lastRegSet1 = await client1.readRegisters();
+  let lastRegSet2 = await client2.readRegisters();
```

Comment:
This is unused, so we we remove it (and possibly also rename `lastRegSet1` to `lastRegSet` or maybe `prevRegSet`)